src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -60,9 +61,12 @@ public HelpWindow() {
         helpBox.initModality(Modality.NONE);
         helpBox.setResizable(true);
         helpBox.getDialogPane().setPrefWidth(900.0);
-        helpBox.getDialogPane().setPrefHeight(530.0);
+        helpBox.getDialogPane().setPrefHeight(450.0);
         helpBox.getDialogPane().getStylesheets().add("/view/DarkTheme.css");
         helpBox.getDialogPane().getStyleClass().add("helpBox");
+        helpBox.getDialogPane().setStyle("-fx-font-family: 'Courier New';");
+        helpBox.setX(200);
+        helpBox.setY(125);
```

Comment:
im not sure about that, but

![screenshot 76](https://cloud.githubusercontent.com/assets/19604232/20049591/f1ea0df8-a4fe-11e6-8d5c-2ea0f615868d.png)

font is not monospaced :/